vm/src/builtins/set.rs

```diff
@@ -997,6 +1029,14 @@ impl AnySet {
     ) -> PyResult<impl std::iter::Iterator<Item = ArgIterable>> {
         Ok(std::iter::once(self.into_iterable(vm)?))
     }
+
+    fn as_inner(&self, _vm: &VirtualMachine) -> PySetInner {
```

Comment:
```suggestion
    fn as_inner(&self) -> &PySetInner {
```
The return type should be fixed to `&PySetInner` for this method.